gem/lib/pupperware/spec_helper.rb

```diff
@@ -54,9 +56,20 @@ def retry_block_up_to_timeout(timeout, &block)
   # Docker Compose Helpers
   ######################################################################
 
+  def docker_compose(command_and_args)
+    overrides = IS_WINDOWS ?
+                  'docker-compose.windows.yml' :
+                  'docker-compose.override.yml'
+    # Only use overrides file if it exists
+    file_arg = File.file?(overrides) ? "--file #{overrides}" : ''
```

Comment:
I'm not too enthusiastic about doing this, but if you need it for now, let's put it in with a comment about hoping to tear it out later.

Also, being explicit like you're doing here might be better for readability, but you really only need to pass `--file docker-compose.yml --file docker-compose.windows.yml` on Windows. `docker-compose.override.yml` is always used automatically without passing any `--file` args. i.e. it can be streamlined to:

```ruby
overrides = IS_WINDOWS && File.file?('docker-compose.windows.yml') ?
  '--file docker-compose.yml --file docker-compose.windows.yml' :
  ''

run_command("docker-compose #{overrides} --no-ansi #{command_and_args}")
```

Reply:
But overall I do like having a `docker_compose` helper centralized!

Reply:
I was thinking that `docker-compose.override.yml` was only used when no `--flag` options were given, but I'm not certain.  i.e. if you're being explicit then it disables implicit behavior

I just want to avoid renaming files prior to execution, like relying on `docker-compose.override.yml` being the default and then `mv`/`cp` files before+after tests.

We can just about the general pattern in meeting today too.